packages/rocketchat-channel-settings/client/views/channelSettings.html

```diff
@@ -47,7 +47,7 @@
 				</div>
 			</div>
 		{{/with}}
-
+		{{#unless isDirectMessage}}
```

Comment:
Can the children of this be indented? That way it's clear they're inside of it..

Reply:
done ;)